Resources/Prototypes/Guidebook/chemicals.yml

```diff
@@ -68,3 +69,10 @@
   name: guide-entry-others
   text: "/ServerInfo/Guidebook/ChemicalTabs/Other.xml"
   filterEnabled: True
+
+  # Funkychem
+- type: guideEntry
+  id: Exotic
+  name: guide-entry-exotic
+  text: "/ServerInfo/Funkystation/Guidebook/ChemicalTabs/Exotic.xml"
+  filterEnabled: True
```

Comment:
I feel like the last category should remain "Others"

Reply:
that's because i did change the order, the order of the guidebook isn't the order of the prototypes in the file